openedx/features/course_experience/utils.py

```diff
@@ -44,7 +48,16 @@ def populate_children(block, all_blocks):
 
         for i in range(len(children)):
             child_id = block['children'][i]
-            child_detail = populate_children(all_blocks[child_id], all_blocks)
+            try:
+                child_detail = populate_children(all_blocks[child_id], all_blocks)
+            except TypeError:
+                if u"MITx+6.002x+MITx_2012_Alumni" in course_outline_root_block[id]:
+                    log.info(u"Child: {child} in block:{block} at index:{index}".format(
```

Comment:
I would suggest adding some keyword which is easier to search in Splunk. e.g. `PopulateChildrenError:`.